projects/frontend/data-pipelines/gui/projects/ui/src/app/app-config.service.ts

```diff
@@ -30,7 +30,12 @@ export class AppConfigService {
         return this.appConfig;
     }
 
+    getSkipAuth(): boolean {
+        return this.appConfig.auth.skipAuth;
+    }
+
     getAuthCodeFlowConfig(): AuthConfig {
+        if (this.getSkipAuth()) return new AuthConfig();
```

Comment:
nit: My personal opinion when we write typescript is to have curly brackets after if statement even if it's without else for better readability, because after transpile to javascript it will undergo minify uglify and will get optimized for Browser execution.
Our main goal should be to make it user friendly as much as possible.
This comment is valid for all places where there are no curly brackets because there is no else.
e.g.

```
if (this.getSkipAuth()) {
    return new AuthConfig();
}

Reply:
nit: I prefer having a one liner in case possible like here, it's shorter and more readable.

Reply:
There is no problem with minifiers on one linears, but I think is more readable having curly brackets especially in a bigger methods that do multiple things.